content/common/view_messages.h

```diff
@@ -174,6 +174,7 @@ IPC_STRUCT_TRAITS_BEGIN(blink::WebScreenInfo)
   IPC_STRUCT_TRAITS_MEMBER(availableRect)
   IPC_STRUCT_TRAITS_MEMBER(orientationType)
   IPC_STRUCT_TRAITS_MEMBER(orientationAngle)
+  IPC_STRUCT_TRAITS_MEMBER(deviceRadius) //[device-radius]
```

Comment:
https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/public/platform/WebScreenInfo.h&sq=package:chromium&type=cs&l=39&rcl=1438687727

First, you need to apply the Blink change to https://github.com/crosswalk-project/blink-crosswalk.